src/test/kotlin/fi/metatavu/timebank/api/test/functional/resources/TestWiremockResource.kt

```diff
@@ -166,6 +166,12 @@ class TestWiremockResource: QuarkusTestResourceLifecycleManager {
                     after = getSixtyDaysAgo()
                 )), 200))
         )
+        wireMockServer.stubFor(
+            get(urlPathEqualTo("/v4/time_registrations"))
+                .inScenario(TIMES_SCENARIO)
+                .whenScenarioStateIs("delete")
```

Comment:
Define the scenario state within the same (if I remember correctly) AbstractTest companion object where all the other scenario states are also defined.